java/ql/src/Security/CWE/CWE-601/UrlRedirect.qhelp

```diff
@@ -16,21 +16,53 @@ controlled by the attacker.</p>
 <p>To guard against untrusted URL redirection, it is advisable to avoid putting user input
 directly into a redirect URL. Instead, maintain a list of authorized
 redirects on the server; then choose from that list based on the user input provided.</p>
-
+<p>
+If this is not possible, then the user input should be validated in some other way,
+for example, by verifying that the target URL is on the same host as the current page.
```

Comment:
```suggestion
If this is not possible, then the user input should be validated in some other way. For example, by verifying that the target URL is on the same host as the current page.
```

Reply:
I'll skip. The existing phrasing is already used elsewhere.